src/components/main.vue

```diff
@@ -95,10 +95,12 @@
       </ResultsList>
 
       <PartialResults />
+    </template>
 
-      <ScrollToTop class="x-button--round" scrollId="mainScroll" :thresholdPx="100">
+    <template #scroll-to-top>
+      <BaseScrollToTop class="x-button x-button--round" scrollId="body-scroll" :thresholdPx="100">
```

Comment:
`x-button` CSS class is already added in the `BaseScrollToTop` component.

```suggestion
      <BaseScrollToTop class="x-button--round" scrollId="body-scroll" :thresholdPx="100">
```

Reply:
Done.